ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -60,6 +60,34 @@ StackLayout {
             }
         }
  
+        Suggestions {
+            id: suggestions
+        }
```

Comment:
`ListModel` is not really static. You can add and remove items from it using `insert`, àppend` and `remove`.  https://doc.qt.io/qt-5/qml-qtqml-models-listmodel.html#methods

You can see how I used that for the EmojiPopup: https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Chat/components/EmojiPopup.qml#L38

Reply:
True that, unfortunately, that won't cut it because the source data is dynamic, not static (the emojis are a static list as well). That's an issue in the filter component as mentioned here: https://github.com/status-im/nim-status-client/pull/562#discussion_r454303448

Reply:
Would creating a dynamic ListModel work on this case? I create a dynamic model here: https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Chat/components/GroupChatPopup.qml#L109-L111
which I fill up as soon as I open a popup: https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Chat/components/GroupChatPopup.qml#L30-L48

I believe something similar could be done for the list of contacts.